hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/ServicesApp.java

```diff
@@ -206,6 +207,8 @@ public interface ServicesApp {
 
     NewSignedStateListener newSignedStateListener();
 
+    Optional<NewRecoveredStateListener> maybeNewRecoveredStateListener();
```

Comment:
why `maybe` ? 🤔

Reply:
Because it's only present if we received `InitTrigger.EVENT_STREAM_RECOVERY`. 

Maybe `optionalNewRecoveredStateListener` is clearer. 😞 But I like `maybe` better!

Reply:
Understood. Thanks